packages/database/src/migrations/20210811034325-AddCOVIDFijiDashboardTestsPerCapita-modifies-data.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+import {
+  insertObject,
+  generateId,
+  findSingleRecord,
+  findSingleRecordBySql,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  code: 'COVID_FJ_Num_Tests_By_Sub_District',
+  builder: 'analyticArithmetic',
+  config: {
+    formula: 'COVIDTest_FJNumTest',
+    aggregation: {
+      COVIDTest_FJNumTest: {
+        type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+        config: {
+          dataSourceEntityType: 'facility',
+          aggregationEntityType: 'sub_district',
+        },
+      },
+    },
+  },
+};
+
+const generateConfig = () => ({
+  code: `FJ_Covid_Tests_Per_100k`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'sub_district',
+          },
+        },
+      ],
+      dataElements: ['population_FJ001', 'COVID_FJ_Num_Tests_By_Sub_District'],
+    },
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'aggregate',
+        organisationUnit: 'drop',
+        period: 'group',
+        population_FJ001: 'sum',
+        COVID_FJ_Num_Tests_By_Sub_District: 'sum',
+      },
+      {
+        transform: 'select',
+        "'Population'": '$row.population_FJ001',
+        "'Tests'": '$row.COVID_FJ_Num_Tests_By_Sub_District',
+        '...': ['period'],
+      },
+      {
+        transform: 'sort',
+        by: '$row.period',
+      },
+      {
+        transform: 'select',
+        "'Population'":
+          'exists($allPrevious.Population) ? subset($allPrevious.Population, index(size($allPrevious.Population))) : -1',
+        '...': ['period', 'Tests'],
+      },
+      {
+        transform: 'filter',
+        where: 'exists($row.Tests) and $row.Population != -1',
+      },
+      {
+        transform: 'select',
+        "'Tests per 100k'": '$row.Tests / ($row.Population * 100000)',
+        "'name'": "periodToDisplayString($row.period, 'DAY')",
+        "'timestamp'": 'periodToTimestamp($row.period)',
+        '...': [],
```

Comment:
```suggestion
```
Shouldn't be necessary I believe 👍

Reply:
Amazing. Could it be possible to love the reports server more